shared/core/src/wasmJsMain/kotlin/com/shub39/grit/core/settings/presentation/ui/section/RootPage.wasmJs.kt

```
@@ -0,0 +1,21 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.shub39.grit.core.settings.presentation.ui.section
+
+import androidx.compose.foundation.lazy.LazyListScope
+
+actual fun LazyListScope.languagePicker(onClick: () -> Unit) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)